Symfony/CS/Console/ConfigurationResolver.php

```diff
@@ -350,8 +355,8 @@ private function resolveConfig()
      */
     private function resolveConfigPath()
     {
-        if (is_file($this->path)) {
-            $this->config->finder(new \ArrayIterator(array(new \SplFileInfo($this->path))));
+        if (count($this->path) === 1 && is_file($this->path[0])) {
+            $this->config->finder(new \ArrayIterator(array(new \SplFileInfo($this->path[0]))));
         } elseif ($this->isStdIn) {
             $this->config->finder(new \ArrayIterator(array(new StdinFileInfo())));
         } elseif (null !== $this->path) {
```

Comment:
I think I understand quit all you explaination but what should I do?

Revert all modifications on `ConfigurationResolver` and call it directly on `FixCommand` for each given path?